web/lib/domain-logic/definitions.js

```diff
@@ -0,0 +1,20 @@
+import { getDb } from './db'
+import { parse } from '@/lib/parser'
+
+const EN_VI_DB = 'en-vi.db'
+
+/**
+ * Get word from db
+ * @param {string} word
+ */
+export async function getDefinitions(word) {
+  const db = await getDb(EN_VI_DB)
+
+  const entry = await db.get('SELECT glossary FROM words WHERE word = :word', {
+    ':word': word,
+  })
+
+  if (entry.glossary && entry.glossary !== '') {
```

Comment:
I think if no record found, `entry` will be undefined? In that case prolly checking `if (entry)` is enough.